packages/@aws-cdk/aws-ecs/lib/log-drivers/journald-log-driver.ts

```diff
@@ -0,0 +1,41 @@
+import { Construct } from '@aws-cdk/core';
+import { ContainerDefinition } from '../container-definition';
+import { BaseLogDriverProps } from './base-log-driver';
+import { LogDriver, LogDriverConfig } from "./log-driver";
+import { stringifyOptions } from './utils';
+
+/**
+ * Specifies the journald log driver configuration options.
+ *
+ * [Source](https://docs.docker.com/config/containers/logging/journald/)
+ */
+export interface JournaldLogDriverProps extends BaseLogDriverProps {}
```

Comment:
I would expect either tsc or tslint to complain about an empty interface here, and you needing to add in a comment to silence the warning. Is that not happening?

Reply:
Yea there is no error when I run tslint.

Would you prefer to use BaseLogDriverProps instead of extending it with an empty interface?